ws/javaee8/docs/list.html

```diff
@@ -97,7 +97,7 @@
 <tr><td colspan="2" class="insidehead">Batch Processing 1.0</td></tr>
 
 <tr>
-<td class="grey1" height="19" width="22%"><a href="../batch/joboperator-api/docs/index.html">The JobOperator API Batch Sample Application</a><br></td>
+<td class="grey1" height="19" width="22%"><a href="mailto:glassfish@javaee.groups.io">The JobOperator API Batch Sample Application</a><br></td>
```

Comment:
Why did you have to change this? The relative path to the index.html is right.

Reply:
I must have accidentally changed it. I'll change it back